sdk/trace/batch_span_processor.go

```diff
@@ -265,7 +265,12 @@ func (bsp *batchSpanProcessor) exportSpans(ctx context.Context) error {
 	}
 
 	if l := len(bsp.batch); l > 0 {
-		global.Debug("exporting spans", "count", len(bsp.batch), "total_dropped", atomic.LoadUint32(&bsp.dropped))
+		global.Debug("exporting spans", "count", len(bsp.batch))
+
+		dropped := atomic.LoadUint32(&bsp.dropped)
+		if dropped > 0 {
```

Comment:
`bsp.dropped` is never reset, so this will result in warning on every export once a span has ever been dropped.  Something like this might work:

```go
dropped := atomic.LoadUint32(&bsp.dropped)
knownDropped := atomic.SwapUint32(&bst.knownDropped, dropped)
if knownDropped < dropped {
  // warn here
}
```